blockstore/badger/blockstore.go

```diff
@@ -80,13 +83,26 @@ const (
 	stateClosed
 )
 
+const (
+	moveStateNone = iota
+	moveStateMoving
+	moveStateCleanup
+	moveStateLock
+)
```

Comment:
1. let's make this an int type
2. godocs on what each of these states means?

Reply:
sure!

Reply:
done.